Python/pylifecycle.c

```diff
@@ -785,7 +791,11 @@ pycore_init_builtins(PyThreadState *tstate)
     PyObject *object__getattribute__ = _PyType_Lookup(&PyBaseObject_Type, &_Py_ID(__getattribute__));
     assert(object__getattribute__);
     interp->callable_cache.object__getattribute__ = object__getattribute__;
-
+    interp->interpreter_trampoline = _Py_MakeTrampoline(
+        INTERPRETER_TRAMPOLINE_CODE, sizeof(INTERPRETER_TRAMPOLINE_CODE), 1, "<interpreter trampoline>");
```

Comment:
See my other comment. `_Py_MakeTrampoline` is pretty tightly coupled to these exact argument values, so they probably just belong there.